porady_odnosnie_studiowania.md

```diff
@@ -348,3 +348,93 @@ Choć w poprzednim punkcie dość dużo wagi położyliśmy na to, żeby nie ba
+  </details>
+
+* **Przygotuj się do obrony/prezentacji**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Ocena pracy dyplomowej składa się z oceny samej pracy oraz oceny jej prezentacji/obrony przed promotorem oraz recenzentem. Dlatego ważne jest, aby dobrze przygotować się do zaprezentowania pracy.
+
+    Koniecznie przyjdź na obronę z własnym sprzętem - laptopem do którego można podłączyć standardowy wydziałowy rzutnik za pomocą kabla HDMI (kabel i rzutnik będą na Ciebie czekały).
+
+    Prezentacja pracy składa się z dwóch części:
+
+    -   właściwej prezentacji podczas której omawiasz pracę (jej problematykę, wykorzystane technologie i rozwiązania etc.) - przygotuj tutaj prezentację multimedialną mającą 10-15 slajdów którą wygłosisz w 10-15 minut. Przećwicz ją przed lustrem/kaczuszką/współlokatorem aby mieć pewność, że potrafisz wygłośić ją płynnie i bez notatek. Prezentacja za krótka bądź za długa negatywnie odbije się na Twojej ocenie
+    -   pytań od recenzenta i promotora - każdy z nich może zadać Ci do trzech pytań związanych z pracą dyplomową
+
+    Ponadto jeżeli w ramach swojej pracy zaimplementowałeś system/algorytm/coś czego działanie możesz pokazać podczas prezentacji pracy, będziesz miał na to do 10 minut między prezentacją a pytaniami od promotora i recenzenta. Upewnij się, że to co chcesz pokazać działa prawidłowo i dla pewności nagraj film (bez audio, komentuj co dzieje się na ekranie na żywo) na wypadek gdyby Twój laptop odmówił posłuszeństwa.
+
+  </details>
+
+* **Możesz odłożyć prezentację pracy**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Jeżeli nie uda Ci się napisać pracy tak, aby złożyć ją w wymaganym terminie, możesz odłożyć jej złożenie i prezentację na następny semestr. Być może będzie się to wiązało ze skreśleniem z listy studentów (patrz dalej), ale nie jest to koniec świata. O zamiarze odłożenia terminu złożenia pracy poinformuj promotora z wyprzedzeniem.
+
+    Ponadto jeżeli termin obrony Twojej pracy inżynierskiej/licencjackiej przypada na semestr letni, możesz prezentować ją na początku września, przed sesją poprawkową. W takim przypadku stracisz możliwość przystąpienia do egzaminu w lipcu i będziesz miał tylko jeden termin podczas sesji poprawkowej.
+  </details>
+
+* **Niezłożenie pracy ma konsekwencje**
+
+  <details>
+    <summary>Szczegóły</summary>
+
+    Jeżeli zaliczyłeś wszystkie wymagania jakie należy zrealizować przed złożeniem pracy (obowiązki, efekty, ECTS etc.), zostaniesz skreślony z listy studentów i będziesz mógł złożyć i obronić pracę w terminie do dwóch lat (cztery semestry) od daty skreślenia. Aby móc złożyć pracę, będziesz musiał złożyć wniosek o przywrócenie na listę studentów (ze względu na chęć złożenia i obrony pracy dyplomowej) na miesiąc-dwa przed terminami ich prezentacji. Uwaga: po skreśleniu z listy studentów stracisz dostęp do niektórych zasobów uniwersyteckich (np. uczelniane konto Microsoft, co może utrudnić komunikację z promotorem).
```

Comment:
Czy termin złożenia wniosku o przywrócenie na listę studentów jest na miesiąc-dwa przed terminami prezentacji pracy? Rok temu kazali mi składać wniosek na kilka tygodni przed rozpoczęciem semestru w którym chciałem się bronić.

Reply:
Osobiście w tym roku (a w zasadzie to 2021/2022) otrzymałem maila z info o konieczności złożenia wniosku 27 maja i zaniosłem go w przeciągu bodajże dwóch tygodni

Reply:
Tak, ale możliwe, że to się zmieniło, albo że miałem złe informacje. Zostawmy to póki co tak jak jest, wrzucę issue żeby to sprawdzić z dziekanatem.